dashboard/test/helpers/levels_helper_test.rb

```diff
@@ -376,16 +376,6 @@ def request
     reset_view_options
   end
 
-  test 'use_google_blockly is true if blocklyVersion is set to Google in view_options even if start_mode is true' do
```

Comment:
Now that we're not treating start mode differently, this test tests an edge case that isn't relevant, I think.